docs/src/pages/components/date-picker/ThemeDatePicker.js

```diff
@@ -17,7 +17,19 @@ const primary = {
   700: '#0059B2',
   800: '#004C99',
   900: '#003A75',
-  1000: '#132F4C',
+};
+
```

Comment:
<img width="355" alt="Screen Shot 2021-07-29 at 17 56 40" src="https://user-images.githubusercontent.com/67129314/127564819-eb61f187-dfd4-49c5-8dee-e1cb3cc7c1c7.png">

Commenting here on the ThemeDatePicker file. There's this big negative, unnecessary space, on it. Do you know why that is? We could remove it.

Reply:
🥲 Not possible at this point, the space is for some month that has date at the bottom row.

<img width="329" alt="Screen Shot 2564-07-30 at 08 55 28" src="https://user-images.githubusercontent.com/18292247/127588131-a1cc6e4c-bcf6-4636-aa7a-4b8cc00d6888.png">

Reply:
Ah, alright, weird and a bummer 😕